src/Compilers/CSharp/Test/Semantic/Semantics/NullableContextTests.cs

```diff
@@ -195,7 +194,7 @@ class Program
             void verify(CSharpParseOptions parseOptions, params string[] expectedAnalyzedKeys)
             {
                 var comp = CreateCompilation(source, parseOptions: parseOptions);
-                comp.NullableAnalysisData = new ConcurrentDictionary<object, NullableWalker.Data>();
+                comp.NullableAnalysisData = new();
```

Comment:
>new(); [](start = 44, length = 6)

Usage of target typed `new` negatively affects readability and I believe we tentatively agreed to not use it in Compilers on an e-mail thread.

Reply:
`NullableAnalysisData` is just a field used by tests, primarily these tests. The actual type of the field seems unimportant.

---
In reply to: [595306179](https://github.com/dotnet/roslyn/pull/51851#discussion_r595306179) [](ancestors = 595306179)